docs/source/development/cdf_guide.rst

```diff
@@ -0,0 +1,677 @@
+.. _cdf_guide:
+
+
+##############################
+Generating CDF Files
+##############################
```

Comment:
Preferably, all underlines should be the same length as the headings for consistency.